src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift

```diff
@@ -58,22 +58,11 @@ final class ResetViewController: UIViewController {
 		header1Label.text = AppStrings.Reset.header1
 		description1TextView.text = AppStrings.Reset.description1
 		discardResetButton.setTitle(AppStrings.Reset.discardButton, for: .normal)
-		discardResetButton.titleLabel?.adjustsFontForContentSizeCategory = true
 
 		infoView.layer.cornerRadius = 14
 		infoTitleLabel.text = AppStrings.Reset.infoTitle
 		infoDescriptionLabel.text = AppStrings.Reset.infoDescription
 
-		guard let resetButton = resetButton, let titleLabel = resetButton.titleLabel else { return }
-
 		resetButton.setTitle(AppStrings.Reset.resetButton, for: .normal)
-		resetButton.titleEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
-		resetButton.backgroundColor = UIColor.preferredColor(for: .brandRed)
-
-		titleLabel.font = UIFont.preferredFont(forTextStyle: .headline)
-		titleLabel.adjustsFontForContentSizeCategory = true
-		titleLabel.lineBreakMode = .byWordWrapping
-
-		resetButton.addConstraint(NSLayoutConstraint(item: resetButton, attribute: .height, relatedBy: .equal, toItem: resetButton.titleLabel, attribute: .height, multiplier: 1, constant: 0))
```

Comment:
This change should be brought back, since remove this line breaks the resizing when accessibility class changes.

These two should be put back:

```swift

guard let resetButton = resetButton, let titleLabel = resetButton.titleLabel else { return }

resetButton.addConstraint(NSLayoutConstraint(item: resetButton, attribute: .height, relatedBy: .equal, toItem: titleLabel, attribute: .height, multiplier: 1, constant: 0))
```

Reply:
This should be handled by the ENAButton itself.

Didn't you mention you would pick this up? It should not be handled on a per view basis.